modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -32,6 +32,8 @@ The API card component shows information about an API, including icon, name, and
 * *Card Border*: Set the thickness of the card borders.
 * *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
 * *Card Shadow*: Set the shadow beneath the card.
+* *API Details Button Label*: Set the label of the API Details button. Default is `API Details`.
```

Comment:
GLOBALLY remove backtick around GUI labels; that is reserved for syntax.

See https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

Reply:
Confusingly, this is the value the user types into the interface, which is why it has backticks. Then the system produces a web page with a button that displays that value as a GUI label.